CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### Bug Fixes
+
+* cli: Update proxy listener to not close when the number of connections left 
+  for the session is zero. The listener will refuse new connections when the 
+  number of connections left is zero but existing connections will be active. 
+  This fixes a CLI client issue where sessions with max connection count 
+  configured were closed when the number connections left hit 0.
+  ([Issue](https://github.com/hashicorp/boundary/issues/4364),
+  ([PR](https://github.com/hashicorp/boundary/pull/4389)) 
+* Safely close websocket connection during session teardown. Fix issue where 
```

Comment:
It was safe before, this makes it sound unsafe 😀 I'd just say that it fixes a spurious error or so